mycroft/api/__init__.py

```diff
@@ -497,19 +497,33 @@ def is_paired(ignore_errors=True):
         # The Mark 1 does perform a restart on RESET.
         return True
 
+    api = DeviceApi()
```

Comment:
this does not appear to be needed here.

Reply:
It's used on the line bellow?

Reply:
must not have been wearing my glasses when I made that comment! :(